src/ui/index.js

```diff
@@ -8,6 +8,13 @@ import UiExports from './ui_exports';
 import UiBundle from './ui_bundle';
 import UiBundleCollection from './ui_bundle_collection';
 import UiBundlerEnv from './ui_bundler_env';
+import i18nPlugin from '../core_plugins/i18n/server/i18n/index';
```

Comment:
Same comment as before, this should come from the server object rather than being imported directly.